src/form-data/ReviewPage.tsx

```diff
@@ -171,6 +172,32 @@ export default function ReviewPage(props: { title: string }) {
             </section>
           );
         })}
+
```

Comment:
Could we consider making this its own component? Design system doesn't seem to have one, but it may be worth housing this markup in its own file.

Reply:
I think i had a brief talk with @nickjg231 on this topic, but I'll create a ticket to move it to it's own exposed component.

Reply:
https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/va-forms-system-core/573

Reply:
We are working on one: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/991

cc @k80bowman